src/transformers/models/gemma3/image_processing_gemma3.py

```diff
@@ -357,16 +358,19 @@ def preprocess(
 
         # All transformations expect numpy arrays.
         images_list = [[to_numpy_array(image) for image in images] for images in images_list]
+        # Search for the first image in the image list.
+        # NOTE: we can't slice the first image with images_list[0][0] if the first batch contains no images. See #36682
+        first_image_in_list = make_flat_list_of_images(images_list)[0]
```

Comment:
Looks a bit like overkill using this function with lots of validations under-the-hood to get the first element. Maybe try this one-liner. What do you think?

```
first_image_in_list = [images for images in images_list if images][0][0]
```